roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -7,6 +7,7 @@ admin_endpoint = https://{{ endpoints.keystone }}:35358/
 
 # Logging #
 log_dir = /var/log/keystone
+default_log_levels=keystone=INFO,oslo.messaging=INFO
```

Comment:
Looking at http://docs.openstack.org/kilo/config-reference/content/keystone-configuration-file.html for `default_log_levels`, `keystone` doesn't look to be one of the options, and `oslo.messaging` is already defaulting to `INFO`. Can you confirm that this setting is actually making a difference to the logging output?